src/components/List.js

```diff
@@ -26,10 +28,47 @@ const List = ({ token }) => {
   };
 
   const handleCheckbox = async (event) => {
-    const currentDate = Date.now(); // changed to using Date.now(), time given in milliseconds
+    const currentDate = Date.now(); // time given in milliseconds
     const queryCollection = await db.collection(token).doc(event.target.id);
+    // filteredListItem is our current snapshot of the doc
+    const filteredListItem = docs.filter(
+      (doc) => doc.id === queryCollection.id,
+    );
```

Comment:
I'm wondering, could you just skip the call the db, and just use the `event.target.id` in place of `queryCollection.id`?  Little less costly, I think.

Reply:
and then below at the `queryCollection.update` part...you could just do:

`db.collection(token).doc(event.target.id).update.....`

Reply:
When we change `queryCollection.id` to `event.target.id`  or just try to console.log the value of `event.target.id`, we get the following error :
<img width="955" alt="target-event-id-error" src="https://user-images.githubusercontent.com/36529777/106970361-8a6d9e00-6701-11eb-92a0-0a68c13d4382.PNG">

Reply:
Hmmm....I guess when I did it when playing around on your branch I grabbed the value of `event.target.id` right at the beginning of the function:

`const id = event.target.id`